docs/semgrep-appsec-platform/wiz.md

```diff
@@ -0,0 +1,68 @@
+---
+slug: wiz
+append_help_link: true
+title: Wiz
+hide_title: true
+description: "Learn how to view Semgrep findings in Wiz's Security Graph."
+tags:
+ - Wiz
+ - Semgrep AppSec Platform
+---
+
+# View Semgrep findings in Wiz's Security Graph
+
+Semgrep allows you to send high severity, high confidence Semgrep Code findings for inclusion in the Wiz Security Graph.
+
+- Semgrep Code only
+- Semgrep sends info from full scans only
+- Semgrep sends info every 24 hours. Choose what gets sent:
+  - Default: Only high severity SAST findings for all repos in your org
+  - Custom: Sends findings for any severity selected and for repos chosen
+- Findings previously sent but not included in the future will be marked as fixed [in Wiz?]
+
+## Prerequisites and requirements
+
+To send Semgrep Code findings to Wiz:
+
+- You must have a license for Wiz Code
+- You must have a Wiz service account that provides Semgrep with the following scopes: `create:external_data_ingestion`, `read:system_activities`, and `read:resources`. If you do not have a service account:
+    1. Create a [Wiz service account](https://docs.wiz.io/wiz-docs/docs/service-accounts-settings?lng=en). When prompted to select the **Type** of the service account, select 
```

Comment:
What type of service account? I'm assuming that most people will already have a service account, but if they don't, what type should they make?

Reply:
Custom integration (GraphQL API)